### DISCUSSION TITLE
Archguard 的领域模型是什么?

### DISCUSSION THREAD

**Discussion by wizardbyron** [archguard/archguard#90] [General]
很想知道 Archguard 自己分析自己会是什么样？
我理想中的工作模式就是符合 DDD 的两关联一循环。

让 Archguard 在本地成为我编程时的架构指南，代码提交后能够及时评估主干上的架构变动，进行评审。

**Comment by phodal** [comment]
上次 Meetup 介绍了这一部分：https://www.bilibili.com/video/BV1Zr4y1G7mA
